mappings/net/minecraft/entity/EntityLike.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_5568 net/minecraft/entity/EntityLike
+	COMMENT A superinterface of {@link Entity}.
```

Comment:
This thing feels like those world interfaces. To me this is like "savable entity" that dedicates to s/l operations